parser/limits.go

```diff
@@ -141,7 +143,7 @@ Options:
 	}
 
 	app := safeGetValue(args, "--app")
-	limits := args["<type>"].([]string)
+	limits := args["<type>=<value>"].([]string)
```

Comment:
Can this command be unchanged and unset both requests and limit as now this command looks similar to set and if user want's to unset an individual request he can do it using the set?